server.js

```diff
@@ -2781,6 +2781,61 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CircleCI build integration.
+// https://circleci.com/api/v1/project/BrightFlair/PHP.Gt?circle-token=0a5143728784b263d9f0238b8d595522689b3af2&limit=1&filter=completed
+camp.route(/^\/circleci\/project\/([^\/]+\/[^\/]+)(?:\/(.*))?\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var userRepo = match[1];  // eg, `doctrine/orm`.
```

Comment:
Please see https://circleci.com/api/v1/project/Doctrine/orm - it seems that this doctrine/orm is not a valid CircleCI project.

```
// Hello human (for machine readable JSON, use the "application/json" Accept header)

{
  "message" : "Project not found"
}
```